crates/iota-node/src/handle.rs

```diff
@@ -135,7 +135,9 @@ impl Drop for IotaNodeHandle {
     fn drop(&mut self) {
         if self.shutdown_on_drop {
             let node_id = self.inner().sim_state.sim_node.id();
-            iota_simulator::runtime::Handle::try_current().map(|h| h.delete_node(node_id));
+            if let Some(h) = iota_simulator::runtime::Handle::try_current() {
```

Comment:
Is that just readability?

Reply:
But we don't care about the map return here, we're just calling a function inside so it's fine?

Reply:
Yes